File: Passby_Value.java
class passByValue {
    public static void passByValue(int x, int y){
        int z=x;
        x=y;
        y=z;
    }
    public static void main(String[] args) {
        int a=5, b=6;
        System.out.println("before swapping a="+""+a+"b="+""+b);
        passByValue(a,b);
        System.out.println("after swapping a="+""+a+"b="+""+b);
    }
}

File: Passby_Value.cpp
#include <iostream>
using namespace std;
void passByValue(int x, int y) //passed by value
{
    int z=x;
    x=y;
    y=z;
}
int main() {
  int a=5 ,b=6;
    cout<<"before swapping"<<endl<<"a:"<<a<<endl<<"b:"<<b<<endl;
    passByValue(a,b);

    cout<<"before swapping"<<endl<<"a:"<<a<<endl<<"b:"<<b<<endl;
    return 0;
}
